src/EnergyPlus/HVACUnitaryBypassVAV.cc

```diff
@@ -2312,87 +2345,63 @@ namespace HVACUnitaryBypassVAV {
                                                                             CBVAV(CBVAVNum).CoolCoilCompIndex,
                                                                             DataHVACGlobals::ContFanCycCoil,
                                                                             HXUnitOn);
-                        if (state.dataLoopNodes->Node(CBVAV(CBVAVNum).DXCoilInletNode).Temp <= CBVAV(CBVAVNum).CoilTempSetPoint) {
-                            //         If coil inlet temp is already below the setpoint, simulated with coil off
-                            PartLoadFrac = 0.0;
-                            HVACHXAssistedCoolingCoil::SimHXAssistedCoolingCoil(state,
-                                                                                CBVAV(CBVAVNum).DXCoolCoilName,
-                                                                                FirstHVACIteration,
-                                                                                DataHVACGlobals::CompressorOperation::Off,
-                                                                                PartLoadFrac,
-                                                                                CBVAV(CBVAVNum).CoolCoilCompIndex,
-                                                                                DataHVACGlobals::ContFanCycCoil,
-                                                                                HXUnitOn);
-                        } else if (state.dataLoopNodes->Node(CBVAV(CBVAVNum).DXCoilOutletNode).Temp < CBVAV(CBVAVNum).CoilTempSetPoint) {
-                            Par(1) = double(CBVAV(CBVAVNum).CoolCoilCompIndex);
-                            Par(2) = CBVAV(CBVAVNum).CoilTempSetPoint;
-                            Par(3) = OnOffAirFlowRatio;
-                            Par(4) = double(CBVAVNum);
-                            if (FirstHVACIteration) {
-                                Par(5) = 1.0;
+                        if (SolFla == -1 && !state.dataGlobal->WarmupFlag) {
```

Comment:
I am not sure what this `SolFla == -1` and subsequent `SolFla == -2` business is but I am sure I don't like it. Do these integers not have symbolic names somewhere?

Reply:
-1 means iteration limit exceeded, -2 means a solution could not be found within starting bounds. No, there are not any symbolic names for this but could easily be added because these (-1 or -2) always mean the same thing.

Reply:
I actually don't recall seeing any symbolic names for these values, which is why they haven't popped up in any of the enum passes.  The SolveRoot/Par/etc. pass is definitely going to be a great win.